website/content/docs/concepts/domain-model/credential-stores.mdx

```diff
@@ -0,0 +1,166 @@
+[project]: /docs/concepts/domain-model/scopes#projects
+
+## Vault Token Requirements
+
+Each Vault credential store must be configured with a unique Vault token.
+The Vault tokens for all credential stores
+must be [periodic][], [renewable][], and an [orphan][].
+All tokens must also have the capabilities of the
+[Vault Boundary Controller Policy][token_policy] described below.
+
+### Vault Policies
+
+The credential store's token must have the capabilities to issue credentials for
+each of it's [credential libraries][] plus the capabilities of the
+[Vault Boundary Controller Policy][token_policy] described below.
+
+We recommend creating a unique Vault policy for each Vault credential store that
+grants the minimum set of permissions needed by the credential store's
+libraries. We also recommend the [Vault Boundary Controller
+Policy][token_policy] be kept distinct from any other Vault policies. This
+allows each credential store to have a token that is configured with two
+polices: one shared by all credential stores and one unique to that credential
+store. This also allows a credential store to be created with the minimum
+permissions necessary at any point in time. The policy unique to a credential
+store can then be updated as needed when credential libraries are added and
+removed from the credential store.
+
+#### Vault Boundary Controller Policy
+
+The token Boundary receives must have the capabilities listed below. An explanation
+for the use of each capability is given.
+
+```hcl
+# Allow Boundary to read and verify the properties of the token. This is
+# provided by the "default" policy.
+path "auth/token/lookup-self" {
+  capabilities = ["read"]
+}
+
+# Allow Boundary to renew the token. This is provided by the "default"
+# policy.
+path "auth/token/renew-self" {
+  capabilities = ["update"]
+}
+
+# Allow Boundary to revoke the token when the credential store is updated
+# to use a new token or the credential store is deleted. This is provided
+# by the "default" policy.
+path "auth/token/revoke-self" {
+  capabilities = ["update"]
+}
+
+# Allow Boundary to renew the credentials in active sessions. This is
+# provided by the "default" policy.
+path "sys/leases/renew" {
+  capabilities = ["update"]
+}
+
+# Allow Boundary to revoke the credentials issued for a session when the
+# session is terminated.
+path "sys/leases/revoke" {
+  capabilities = ["update"]
+}
+
+# Allow Boundary to read and verify the token's capabilities for each Vault
+# path used by the credential store. This is provided by the "default"
+# policy.
+path "sys/capabilities-self" {
+  capabilities = ["update"]
+}
+```
+
+The above [`boundary-controller` policy](/data/vault/boundary-controller-policy.hcl) is
+available for download. Below is an example of writing this policy to Vault:
+
+```shell-session
+# Download the policy
+$ curl https://boundaryproject.io/data/vault/boundary-controller-policy.hcl -O -s -L
```

Comment:
Love the fact that we give the policy here as a downloadable link :)

Reply:
ty!